translations/en.json

```diff
@@ -2733,7 +2733,7 @@
   "use-keycard-subtitle": "Keys will be stored on your Keycard",
   "use-photo": "Use Photo",
   "use-recovery-phrase": "Use recovery phrase",
-  "use-the-multichain-wallet": "Use the leading multi-chain self-custodial wallet",
+  "use-the-multichain-wallet": "Enjoy our multi-chain self-custodial wallet",
```

Comment:
FYI: I asked in [#design-discussion](https://discord.com/channels/1210237582470807632/1274068685266489434/1313570627675881512) about using `your` instead of `our`.

Reply:
Thank you, updated

Reply:
Thanks @Parveshdhull!